applications/rtl_lan9252demo/slave.c

```diff
@@ -155,7 +155,7 @@ void DIG_process (void)
 
 /********** TODO: Generic code beyond this point ***************/
 
-static const char *spi_name = "/spi1/lan9252";
+static const char *spi_name = "/dev/lan9252";
```

Comment:
The device driver node is created as /dev/lan9252.
![1](https://user-images.githubusercontent.com/10510127/28055218-fb1a2244-6652-11e7-86a0-f7755a293c8f.png)